database_schemas/cassandra/benchflow.cql

```diff
@@ -87,7 +87,7 @@ CREATE TABLE process (
   to_ignore boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), source_process_instance_id, process_instance_id)
+  PRIMARY KEY ((experiment_id, trial_id), process_instance_id)
```

Comment:
@VincenzoFerme It was all NULL in the dump I was using for testing, so beats me. The process instance id though is always present